kaldb/src/main/java/com/slack/kaldb/server/ArmeriaService.java

```diff
@@ -113,7 +117,20 @@ public Builder withGrpcService(BindableService grpcService) {
               // if not using the client timeout header - separate, lower timeouts
               // should be configured for indexer / cache nodes than that of the query server
               .useClientTimeoutHeader(true)
-              .useBlockingTaskExecutor(true);
+              .useBlockingTaskExecutor(true)
+              .intercept(
+                  new ServerInterceptor() {
+                    @Override
+                    public <ReqT, RespT> ServerCall.Listener<ReqT> interceptCall(
+                        ServerCall<ReqT, RespT> call,
+                        Metadata headers,
+                        ServerCallHandler<ReqT, RespT> next) {
+                      // sets gzip message encoding per
```

Comment:
Please add a comment this is for response side encryption.

Reply:
Done.